### DISCUSSION TITLE
Xcaddy build in x86

### DISCUSSION THREAD

**Discussion by StephenMiracle** [php/frankenphp#631] [Q&A]
I couldn't ever get xcaddy to run the custom build in x86 environment. I was able to get it to compile, but even then it was throwing errors when running. I ended up just creating an arm64 graviton on AWS to get it to build and run.

We may want to put a note about x86 in the "known issues" as problematic for custom builds.

*note:* I didn't try it again after adding the cbrotli module, but I don't think it was related.

**Comment by dunglas** [comment]
@StephenMiracle That's surprising because we build for x86 in the CI and provide x86 Docker images.
